miniupnpc/CMakeLists.txt

```diff
@@ -96,6 +96,9 @@ if (WIN32)
 #  find_library (NSL_LIBRARY NAMES nsl)
 #  find_library (RESOLV_LIBRARY NAMES resolv)
 #  set (LDLIBS ${SOCKET_LIBRARY} ${NSL_LIBRARY} ${RESOLV_LIBRARY} ${LDLIBS})
+elseif (HAIKU)
+  list(APPEND LDLIBS -lnetwork)
```

Comment:
It seems cleaner (I think) to do the same as the Solaris code above?

    find_library (SOCKET_LIBRARY NAMES network)
    set (LDLIBS ${SOCKET_LIBRARY})

(if hat still work, this code is commented, so maybe not)

Reply:
I'll try to experiment with this.